deps/ntlmclient/ntlm.c

```diff
@@ -47,11 +47,9 @@ ntlm_client *ntlm_client_init(ntlm_client_flags flags)
 {
 	ntlm_client *ntlm = NULL;
 
-	if ((ntlm = malloc(sizeof(ntlm_client))) == NULL)
+	if ((ntlm = calloc(1, sizeof(ntlm_client))) == NULL)
```

Comment:
It would probably make sense to split out all changes to ntlmclient into a separate commit so that @ethomson can easily cherry-pick them to upstream.

Reply:
Now live at #5571